fpm/src/fpm_command_line.f90

```diff
@@ -449,13 +464,26 @@ subroutine get_command_line_settings(cmd_settings)
     contains
 
     subroutine check_build_vals()
+        character(len=:), allocatable :: flags
 
         val_compiler=sget('compiler')
         if(val_compiler.eq.'') then
             val_compiler='gfortran'
         endif
 
-        val_build=trim(merge('release','debug  ',lget('release')))
+        val_flag = " " // sget('flag')
+        val_profile = sget('profile')
+        if (val_flag == '') then
+            call get_default_compile_flags(val_compiler, val_profile == "release", val_flag)
+        else
+            select case(val_profile)
+            case("release", "debug")
```

Comment:
All the above calls to `set_args` say `--profile "default"`, but here and below in the help you use say debug. We should be consistent. Probably debug for now, since previously we said release vs debug in the help message.

Reply:
The default profile is a placeholder, maybe it should be an empty string instead. The difference between default and debug is that the latter appends flags while the former yields against flags.